CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Set `availabilityZoneSubnetType` to `private` for machine pools. This prevents nodes from landing on public subnets.
```

Comment:
I think this needs a breaking change flag as this will create a new NodeGroup and new ASG in AWS and destroy the old one

Reply:
Done